api/controllers/tasks.py

```diff
@@ -120,6 +124,33 @@ def process_proposal(credentials, tpid):
             last_updated=db.sql.func.now(),
         )  # Annotation config is sentiment example.
 
+        mySMTP = smtplib.SMTP("smtp.gmail.com", 587)
```

Comment:
I think there is a method `mail.send` that you might be able to use instead of doing all this manually. You import `import common.mail_service as mail` and then fill in a subject, body, etc. and it handles the sending portion you.

There is an example in https://github.com/facebookresearch/dynabench/blob/main/api/controllers/users.py#L221